src/main/java/ch/njol/skript/expressions/ExprNamed.java

```diff
@@ -35,60 +35,84 @@
 import ch.njol.skript.expressions.base.PropertyExpression;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.Literal;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.util.Getter;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Named Item/Inventory")
-@Description("Directly names an item/inventory, useful for defining a named item/inventory in a script. " +
-		"If you want to (re)name existing items/inventories you can either use this expression or use <code>set <a href='#ExprName'>name of &lt;item/inventory&gt;</a> to &lt;text&gt;</code>.")
-@Examples({"give a diamond sword of sharpness 100 named \"&lt;gold&gt;Excalibur\" to the player",
-		"set tool of player to the player's tool named \"&lt;gold&gt;Wand\"",
-		"set the name of the player's tool to \"&lt;gold&gt;Wand\"",
-		"open hopper inventory named \"Magic Hopper\" to player"})
+@Description({
+	"Directly names an item/inventory, useful for defining a named item/inventory in a script.",
+	"If you want to (re)name existing items/inventories you can either use this expression " +
+			"or use <code>set <a href='#ExprName'>name of &lt;item/inventory&gt;</a> to &lt;text&gt;</code>."
+})
+@Examples({
+	"give a diamond sword of sharpness 100 named \"&lt;gold&gt;Excalibur\" to the player",
+	"set tool of player to the player's tool named \"&lt;gold&gt;Wand\"",
+	"set the name of the player's tool to \"&lt;gold&gt;Wand\"",
+	"open hopper inventory named \"Magic Hopper\" to player"
+})
 @Since("2.0, 2.2-dev34 (inventories)")
 public class ExprNamed extends PropertyExpression<Object, Object> {
```

Comment:
IMO some tests would be good. You'd be fine to incorporate the tests from <https://github.com/SkriptLang/Skript/pull/6159>